diff -Naur mig_36_1/user_design/rtl/ddr2_infrastructure.vhd mig_36_1_ddr2/user_design/rtl/ddr2_infrastructure.vhd
--- mig_36_1/user_design/rtl/ddr2_infrastructure.vhd	2011-09-22 11:29:57.000000000 +0200
+++ mig_36_1_ddr2/user_design/rtl/ddr2_infrastructure.vhd	2011-09-22 16:02:13.000000000 +0200
@@ -104,7 +104,8 @@
     rst0            : out std_logic;
     rst90           : out std_logic;
     rst200          : out std_logic;
-    rstdiv0         : out std_logic
+    rstdiv0         : out std_logic;
+    clkinp          : out std_logic
     );
 end entity ddr2_infrastructure;
 
@@ -166,6 +167,7 @@
   clk90   <= clk90_bufg;
   clk200  <= clk200_bufg;
   clkdiv0 <= clkdiv0_bufg;
+  clkinp <= sys_clk_ibufg;
 
   DIFF_ENDED_CLKS_INST : if(CLK_TYPE = "DIFFERENTIAL") generate
   begin
@@ -220,7 +222,7 @@
   end generate;
 
   NOCLK200_CHECK_GND: if ( (NOCLK200 = true) and (CLK_TYPE = "SINGLE_ENDED")) generate
-     clk200_bufg <= '0';
+     clk200_bufg <= idly_clk_200;
   end generate;
 
 
@@ -233,7 +235,7 @@
     u_pll_adv: PLL_ADV
       generic map (
         BANDWIDTH          => "OPTIMIZED",
-        CLKIN1_PERIOD      => CLK_PERIOD_NS,
+        CLKIN1_PERIOD      => CLK_PERIOD_NS*2.0,
         CLKIN2_PERIOD      => 10.000,
         CLKOUT0_DIVIDE     => CLK_PERIOD_INT,
         CLKOUT1_DIVIDE     => CLK_PERIOD_INT,
@@ -255,7 +257,7 @@
         CLKOUT5_DUTY_CYCLE => 0.500,
         COMPENSATION       => "SYSTEM_SYNCHRONOUS",
         DIVCLK_DIVIDE      => 1,
-        CLKFBOUT_MULT      => CLK_PERIOD_INT,
+        CLKFBOUT_MULT      => CLK_PERIOD_INT*2,
         CLKFBOUT_PHASE     => 0.0,
         REF_JITTER         => 0.005000
         )
diff -Naur mig_36_1/user_design/rtl/mig_36_1.vhd mig_36_1_ddr2/user_design/rtl/mig_36_1.vhd
--- mig_36_1/user_design/rtl/mig_36_1.vhd	2011-09-22 11:29:57.000000000 +0200
+++ mig_36_1_ddr2/user_design/rtl/mig_36_1.vhd	2011-09-22 14:19:51.000000000 +0200
@@ -195,7 +195,9 @@
    ddr2_dqs              : inout  std_logic_vector((DQS_WIDTH-1) downto 0);
    ddr2_dqs_n            : inout  std_logic_vector((DQS_WIDTH-1) downto 0);
    ddr2_ck               : out   std_logic_vector((CLK_WIDTH-1) downto 0);
-   ddr2_ck_n             : out   std_logic_vector((CLK_WIDTH-1) downto 0)
+   ddr2_ck_n             : out   std_logic_vector((CLK_WIDTH-1) downto 0);
+   clk100 		 : out   std_ulogic;
+   clkdiv		 : out   std_ulogic
    );
 
 end entity mig_36_1;
@@ -240,17 +242,18 @@
       clk200_p             : in    std_logic;
       clk200_n             : in    std_logic;
       idly_clk_200         : in    std_logic;
-      sys_rst_n            : in    std_logic;
-      rst0                 : out   std_logic;
-      rst90                : out   std_logic;
-      rstdiv0              : out   std_logic;
-      rst200               : out   std_logic;
       clk0                 : out   std_logic;
       clk90                : out   std_logic;
-      clkdiv0              : out   std_logic;
       clk200               : out   std_logic;
-      idelay_ctrl_rdy      : in    std_logic
+      clkdiv0              : out   std_logic;
 
+      sys_rst_n            : in    std_logic;
+      idelay_ctrl_rdy      : in    std_logic;
+      rst0                 : out   std_logic;
+      rst90                : out   std_logic;
+      rst200               : out   std_logic;
+      rstdiv0              : out   std_logic;
+      clkinp          : out std_logic
       );
   end component;
 
@@ -473,7 +476,8 @@
       clk90                 => clk90,
       clkdiv0               => clkdiv0,
       clk200                => clk200,
-      idelay_ctrl_rdy       => idelay_ctrl_rdy
+      idelay_ctrl_rdy       => idelay_ctrl_rdy,
+      clkinp                => clk100
    );
 
   u_ddr2_top_0 : ddr2_top
